src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -179,27 +332,38 @@ static void OnFlyoutBehaviorChanged(BindableObject bindable, object oldValue, ob
 			}
 		}
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='BackgroundColorProperty']/Docs/*" />
+		/// <summary>
+		/// BackgroundColor defines the background color in the Shell chrome. 
+		/// The color won't fill in behind the Shell content.
+		/// </summary>
 		public static readonly new BindableProperty BackgroundColorProperty =
 			BindableProperty.CreateAttached("BackgroundColor", typeof(Color), typeof(Shell), null,
 				propertyChanged: OnShellAppearanceValueChanged);
 
-		/// <summary>Bindable property for attached property <c>DisabledColor</c>.</summary>
+		/// <summary>
+		/// Defines the color to shade text and icons that are disabled.
+		/// </summary>
 		public static readonly BindableProperty DisabledColorProperty =
 			BindableProperty.CreateAttached("DisabledColor", typeof(Color), typeof(Shell), null,
 				propertyChanged: OnShellAppearanceValueChanged);
 
-		/// <summary>Bindable property for attached property <c>ForegroundColor</c>.</summary>
+		/// <summary>
+		/// Defines the color to shade text and icons.
+		/// </summary>
 		public static readonly BindableProperty ForegroundColorProperty =
 			BindableProperty.CreateAttached("ForegroundColor", typeof(Color), typeof(Shell), null,
 				propertyChanged: OnShellAppearanceValueChanged);
 
-		/// <summary>Bindable property for attached property <c>TabBarBackgroundColor</c>.</summary>
+		/// <summary>
+		/// Defines the background color for the tab bar. If the property is unset, the BackgroundColor property value is used.
```

Comment:
Missing crefs for the properties being references. This applies to most colors.

Reply:
Applied the missing cref here and in the rest of color related properties.